packages/vscode/.vscodeignore

```diff
@@ -0,0 +1,28 @@
+../../node_modules/**
```

Comment:
Should we consider ignoring everything at the top of `.vscodeignore`, and then allow-listing what we need? The Zowe Explorer `.vscodeignore` does this: https://github.com/zowe/vscode-extension-for-zowe/blob/main/packages/zowe-explorer/.vscodeignore

I think it is good practice to structure the file that way 🙂 It makes the file easier to read since our allow list is shorter than our block list. And when we add new files in the future that we don't need in the VSIX, it reduces the risk of forgetting to add them to the ignore list because we don't have to.

Reply:
I will give this a test, I remember reading a[n issue on how this no longer works](https://github.com/microsoft/vscode-vsce/issues/576), but may be worth testing and checking with `vsce ls` to see what's being not ignored.

Reply:
Just tested, and it seems like it no longer works by ignoring everything else and then adding what we do not want to ignore, it seems like `.vscodeignore` is now following the [rules](https://git-scm.com/docs/gitignore) from `.gitignore`